include/config/config_objects.h

```diff
@@ -27,7 +27,7 @@
 /**
  * Allow previewing blue coin locations from a blue coin switch when Mario is standing on the blue coin switch.
  */
-// #define BLUE_COIN_SWITCH_PREVIEW
+#define BLUE_COIN_SWITCH_PREVIEW
```

Comment:
This one feels like a bit of a wild card to enable by default compared to everything else here

Reply:
thats fair i can disable it